test/edu-benefits/5490/schema/minimal-test.json

```diff
@@ -0,0 +1,96 @@
+{
+  "privacyAgreementAccepted": true,
+  "submission": {
```

Comment:
Why does this file have a "submission" field but the maximal example doesn't? Trying to follow suit for the 1990e schema tests in https://github.com/department-of-veterans-affairs/vets-website/tree/1990e-schema-test-rr